Apps/W1/HybridGP/app/src/Migration/History/AI/GPPopulateHistTables.Codeunit.al

```diff
@@ -0,0 +1,892 @@
+codeunit 40900 "GP Populate Hist. Tables"
```

Comment:
I would not put this code unit under the AI heading as I am not sure it fits the AI definition. I would suggest just under History.

Reply:
I thought that was AL... I'll move it, thanks.